amplify/backend/api/petrakipApi4data/schema.graphql

```diff
@@ -0,0 +1,95 @@
+# General Types and Enums
+
+type S3Object {
+  bucket: String!
+  key: String!
+  region: String!
+}
+
+enum MimeType {
+  image
+  video
+  text
+  audio
+}
```

Comment:
Wollen wir das wirklich MimeType nennen 😮 ? Normal ist wäre dann ja application/image oder sowas

Reply:
@maxhaensel just telling it is bad or should we really name it like that or do this is not a good review or feedback. If you think we should rename the type, please make suggestions.

Reply:
ContentType seems more appropriate, good suggestion